src/CellData.jl

```diff
@@ -320,3 +320,35 @@ end
 CellData.jump(a::DistributedCellField) = DistributedCellField(map_parts(jump,a.fields))
 CellData.jump(a::SkeletonPair{<:DistributedCellField}) = a.⁺ + a.⁻
 CellData.mean(a::DistributedCellField) = DistributedCellField(map_parts(mean,a.fields))
+
+
+# DistributedCellDof
+
+struct DistributedCellDof{A,B} <: DistributedCellDatum
+  dofs::A
+  metadata::B
+  function DistributedCellDof(dofs::AbstractPData{<:CellDof},metadata=nothing)
+      A = typeof(dofs)
+      B = typeof(metadata)
+      new{A,B}(dofs,metadata)
+  end
+end
+
+local_views(s::DistributedCellDof) = s.dofs
+
+(a::DistributedCellDof)(f) = evaluate(a,f)
+
+function Gridap.Arrays.evaluate!(cache,s::DistributedCellDof,f::DistributedCellField)
+  map_parts(local_views(s),local_views(f)) do s, f
+      evaluate!(nothing,s,f)
```

Comment:
why the first argument to `evaluate!` is hard-coded to `nothing`? Are we sure that `return_cache(s,f)` returns nothing?

Reply:
The function we are wrapping is located in `Gridap.jl/src/CellData/CellDofs.jl`: 
```
function evaluate!(cache,s::CellDof,f::CellField)

  trian_f = get_triangulation(f)
  trian_s = get_triangulation(s)

  if trian_f !== trian_s
    @unreachable """\n
    A CellDof can only be evaluated on a CellField defined on the same Triangulation.
    """
  end

  b = change_domain(f,s.domain_style)
  lazy_map(evaluate,get_data(s),get_data(b))
end
```

It returns a lazy map and does not use the cache.